app/notes/[id]/page.tsx

```diff
@@ -18,6 +17,16 @@ export default async function Page({ params }: any) {
         const note = await getNote(params.id);
         console.log("Note:", note);
 
+        async function deleteNote() {
```

Comment:
use server directive usually should never be used in a function, it signifies some bad practice, at least as far as I remember, try looking it up

Reply:
i saw it in a tutorial about server actions. i should have implemented the action in an action.js file